PusherMainView/PusherMainView/PushTypesViewController.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import APNS
+
+final class PushTypesViewController: NSViewController {
+    @IBOutlet private var tableView: NSTableView!
+    var pusherStore: PusherInteracting?
+    private let pushTypes: [String]
+    private let cellIdentifier = "PushTypeTableCellView"
+
+    required init?(coder: NSCoder) {
+        guard let pushTypes = PlistFinder<[String]>.model(for: "pushtypes", and: PushTypesViewController.self) else {
+            return nil
+        }
+
+        self.pushTypes = pushTypes
+
+        super.init(coder: coder)
+        #if DEBUG
+        print("\(self.className) init")
+        #endif
+    }
+
+    deinit {
+        #if DEBUG
+        print("\(self.className) deinit")
+        #endif
+    }
+
+    // MARK: - Life Cycle
+
+    override func viewDidAppear() {
```

Comment:
I think we can remove those 2 functions as they only call `super`

Reply:
Done.